book/website/community-handbook/onboarding-calls.md

```diff
@@ -0,0 +1,38 @@
+(ch-onboarding-calls)=
+# Onboarding Calls
```

Comment:
I think this call may be in the wrong location! For some reason it shows up for me under the newsletter section?

Reply:
Not the case for me - see the screenshot of the preview - can you share what you are seeing? 
<img width="333" alt="image" src="https://github.com/user-attachments/assets/e058a0d8-e793-4215-91f1-e81a1367b41c" />